### ISSUE TITLE
[Request] API for macros support

### ISSUE THREAD

**Issue by paulo-roger** [Octarines/FoundryHourglass#10]
If possible I would like to be able to call each different preset I have set or even set new ones through macros.
Also if could have a function to control the manual advance of the sand, like `timeadvance(integer)`

I appreciate your work with this module anyway.

**Comment by Octarines** [comment]
Thanks for the suggestion. I must admit I haven't even looked at the APIs for the macros. I'll do a little research and see what is possible.

**Comment by bearsy1** [comment]
Ooooh, I second this idea. It would be neat to also have macro support (to save a few clicks if nothings else hahaha). <3

**Comment by Octarines** [comment]
Sorry this was a long time coming but I have now released v1.9.0 of Hourglass which has macro support for foundry v12. 
It supports creation and closing of both default and preset timers.
I have opted to omit macro incrementation of the manual timers for now as each timer window has its own instance id that is new each time you create one. Incrementing would need to be able to target this window by id so seemed potentially a bit too involved for a macro. If you have further suggestions I would love to hear from you!
